compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -187,6 +187,9 @@ static const char *getDefaultEnabledUkernels(Attribute attr) {
   if (isAArch64(targetAttr)) {
     return "mmt4d";
   }
+  if (isRISCV64(targetAttr)) {
+    return "mmt4d";
```

Comment:
Let's keep the defaults switch for a separate PR. Better iterate on the new ukernels in this PR, then later once that has settled change the defaults. In the interim, you can manually pass the ukernel flag/attribute.